selenium-tests/test_registration.py

```diff
@@ -87,30 +116,65 @@ def run(self, driver):
         assert self.expected_page in driver.current_url
 
 
+def fill_and_submit_stripe_form(driver):
+    """
+    This function assumes you use a unique email address, if not, Stripe will try and verify you with
+    an SMS message code, and we don't handle that case yet.
+    """
+    pay_button_xpath = "//form[@class='stripe']/button[@class='stripe-button-el']"
+    button = driver.find_element_by_xpath(pay_button_xpath)
+    button.click()
+
+    if not is_mobile(driver):
+        # mobile stripe doesn't use an iframe, just opens a new tab
+        stripe_iframe = driver.find_element_by_xpath("//iframe[@name='stripe_checkout_app']")
+        driver.switch_to_frame(stripe_iframe)
+    else:
+        driver.switch_to.window("stripe_checkout_tabview")
+
+    # we can't just send the entire card# for some reason, send it in 4 chunks.
+    for i in range(4):
+        driver.find_element_by_id('card_number').send_keys(STRIPE_FAKE_CREDIT_CARD)
+
+    for part in STRIPE_FAKE_EXP_DATE.split('/'):
+        driver.find_element_by_id('cc-exp').send_keys(part)
+
+    driver.find_element_by_id('cc-csc').send_keys(STRIPE_FAKE_CSC)
+    zip_elem = driver.find_element_by_id('billing-zip')
+    zip_elem.send_keys(STRIPE_FAKE_ZIP)
+    zip_elem.submit()
+
+    if not is_mobile(driver):
+        driver.switch_to_default_content()
+
+    time.sleep(4)  # hax.
+
+
 def generate_unique_email():
     return "testemail-{}@testemail.com".format(str(uuid.uuid4()))
 
+
 birthdate_elements = ["jq-dte-month", "jq-dte-day", "jq-dte-year"]
 
 # order matters
 regform_validations = [
-    TextFieldValidator(field_name="first_name"),
-    TextFieldValidator(
+    InputFieldValidator(field_name="first_name"),
+    InputFieldValidator(
         expected_error="First Name is a required field",
```

Comment:
I know this is unlikely, but is there any way these errors could be pulled from the actual code rather than hardcoded? It seems like a bit of a pain to update text in two places.